src/dmd/dsymbolsem.d

```diff
@@ -1844,6 +1844,61 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
 
     override void visit(PragmaDeclaration pd)
     {
+        StringExp verifyMangleString(ref Expression e)
+        {
+            auto se = semanticString(sc, (*pd.args)[0], "mangled name");
+            if (!se)
+                return null;
+            (*pd.args)[0] = se; // Will be used later
```

Comment:
Having the side effect in a `verify` function is a bit misleading, especially since it returns an unused value. Either return `bool` or use the return value.

Reply:
done

Reply:
Ah, I remember why that doesn't work, that comment is there for a reason. otherwise `fail_compilation/ice13788.d` will trip. undoing. done now through a ref assignment

Reply:
Is it because of the eager assignment to `(*pd.args)[0]` ? If so the code suggestion I posted fixes that.

Reply:
The value is unused in _this_ PR it is used in the other one.